src/app/docs/datepicker/datepicker-docs.component.html

```diff
@@ -7,14 +7,107 @@
     The datepicker module provides an input and calendar for users to select dates or fuzzy dates.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-code-examples
+  <sky-docs-demo
+    [heading]="'Standard datepicker demo'"
+  >
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include date range validator (01/01/2020 - 01/31/2020)"
```

Comment:
On the standard datepicker demo, if I select this checkbox without ever having clicked in the demo, it doesn't take immediate effect. However, if I select the field in the demo before clicking the checkbox (even if I subsequently place focus elsewhere), the field is immediately highlighted with the red error border. The fuzzy datepicker demo on the other hand highlights the field with the red error border when the checkbox is selected regardless of whether the field was ever selected. Is this behavior expected?

Reply:
Looks like thats an issue with the datepicker itself. I'll file an issue separately for that.